pages/docs/integrations/grouparoo-dbt.mdx

```diff
@@ -0,0 +1,104 @@
+
+## Create a Grouparoo Project Within your dbt Project
+
+To integrate Grouparoo within your existing dbt project, you'll want to follow our installation docs and make sure you initiate your Grouparoo project in the same directory as your
+dbt project.
+
+<PluginDocsInstallation plugin="dbt" />
+
+<Alert variant="warning">
+  Note that the main purpose of the dbt Plugin is to enable shared configuration
+  of your dbt project. You will also need to install the Grouparoo Plugin for
+  your data warehouse.
+</Alert> Grouparoo's dbt plugin can support the following data warehouses:
+
+- [BigQuery](/docs/integrations/grouparoo-bigquery)
+- [Postgres](/docs/integrations/grouparoo-postgres)
+- [Redshift](/docs/integrations/grouparoo-redshift)
+- [Snowflake](/docs/integrations/grouparoo-snowflake)
+
+## Orchestrating Grouparoo with dbt
+
+You can use an [App Refresh Query](/docs/getting-started/product-concepts#app-refresh-query) to orchestrate your Grouparoo Project based on dbt runs. An App Refresh Query checks for new data in a specific place in your warehouse as often as every minute. If new data is found, it will trigger all Schedules associated with that App. To get started with App Refresh Query, you'll want to make sure you have a single place for Grouparoo to check for data.
+
+### Step 1: Set up a meta table
+
+<Alert variant="success">
+  This step is only required if you would like to generate a new `meta` table
+  for Grouparoo to check for completed runs.
+</Alert>
+
+You can use dbt's [`on-run-end` hook](https://docs.getdbt.com/reference/project-configs/on-run-start-on-run-end) to generate a new table that is updated upon the completion of each dbt run.
+
+At the very simplest, you can do this by adding a macro to your project that generates a table that is updated after each run. Add the following line to your `dbt_project.yml`:
+
+```yml
+on-run-end: "{{ on_run_end(results) }}"
+```
+
+Then in the `/macros` directory, create a new file called `on_run_end.sql`. The following will generate a `dbt_meta` table if it doesn't already exist and will keep track of how many namespaces had success, errors, or were skipped.
+
+```sql
+{% macro on_run_end(results) %}
+  {% set ns = namespace(success=0, error=0, skipped=0) %}
+
+  {% for res in results %}
+    {{ log("result: " ~ res.status ~ " node: " ~ res.node.unique_id ~ " message: " ~ res.message, info=True) }}
+    {% if res.status == "success" %}
+      {% set ns.success = ns.success + 1 %}
+    {% elif res.status == "error" %}
+      {% set ns.error = ns.error + 1 %}
+    {% elif res.status == "skipped" %}
+      {% set ns.skipped = ns.skipped + 1 %}
+    {% endif %}
+  {% endfor %}
+
+  {{ log("on_run_end  success: " ~ ns.success ~ " error: " ~ ns.error ~ " skipped: " ~ ns.skipped, info=True) }}
+  {% if execute %}
+    CREATE TABLE IF NOT EXISTS dbt_meta (last_run_at TIMESTAMP, success INT, error INT, skipped INT);
```

Comment:
It might be nice to make this look even simpler. Above, we could put:

> on-run-end: "CREATE TABLE IF NOT EXISTS dbt_meta (last_run_at TIMESTAMP); INSERT INTO dbt_meta (last_run_at) VALUES (NOW());"

And then link to the dbt sample project like: "Or you can use a macro that records more information as seen here<>"